modules/core/src/main/java/org/apache/ignite/internal/cdc/CdcMain.java

```diff
@@ -183,6 +185,29 @@ public class CdcMain implements Runnable {
     private static final IgniteBiPredicate<WALRecord.RecordType, WALPointer> ACTIVE_RECS =
         (type, ptr) -> type == DATA_RECORD_V2 || type == CDC_DATA_RECORD;
 
+    /** Histogram buckets for duration wal processing in nanoseconds. */
+    public static final long[] HISTOGRAM_BUCKETS = new long[] {
+        NANOSECONDS.convert(1, MILLISECONDS),
+        NANOSECONDS.convert(10, MILLISECONDS),
+        NANOSECONDS.convert(100, MILLISECONDS),
+        NANOSECONDS.convert(250, MILLISECONDS),
+        NANOSECONDS.convert(1000, MILLISECONDS)
+    };
+
+    /** */
+    public static final String EVENTS_CONSUMPTION_TIME = "EventsConsumptionTime";
+
+    /** */
+    public static final String EVENTS_CONSUMPTION_TIME_DESC =
```

Comment:
Fix it in the same way as the documentation.

Reply:
Done!